### PR TITLE
try two

### PR DESCRIPTION
**Pull request by JennieDalgren**
Bodu body body test ny pr template

### CODE REVIEW

**Comment by JennieDalgren** [comment]
test

File: code/index.html [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+  <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet" />
+</head>
+
+<body>
+  <h1>Jennies Javascript Pizzeria</h1>
+  <p>Logic is executed automatically</p>
+  <p>TRYING!!!!!!!</p>
+  <script src="./script.js"></script>
```
Comment (HIPPIEKICK) [inline code/index.html:18]: This is an inline comment of row 18

**Review by HIPPIEKICK — APPROVED** [review]
This is an overall comment of project-pizza

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
You need to do some changes

**Pull request closed without merging**